pyroengine/pi_utils/requirements.txt

```diff
@@ -0,0 +1 @@
+gpiozero>=1.5.1
```

Comment:
Does `psutil` package has to be added in this requirements file  ?

Reply:
Or more precisely, is this psutil package used elsewhere than in pi_utils in the repo? 
If not, it should be better to require it here and not in the general requirements file 

(Thinking about this point because, some of pi_utils scripts will be used on pi camera, hence, these are some of the requirements for the pi camera)

Reply:
well might be the same for `pandas` if not used elsewhere

Reply:
psutil is already in the package requierements so no need to add it twice. We need it to be installed for testing, this is way i moved it in package requirements. Idem for pandas

Reply:
Indeed ! 

Not blocking for the PR, but, would it be okay to still add those libs here ? Would be redondant, but we would exactly know what is needed for picameras .    Just an open question, we might have other solutions for a nice set up :)